src/Engines/thinEngine.ts

```diff
@@ -171,6 +171,29 @@ export class ThinEngine {
         Effect.ShadersRepository = value;
     }
 
+     /**
+     * Gets or sets the textures that the engine should not attempt to load as compressed
+     */
+    public static ExcludedCompressedTextures: string[] = [];

```

Comment:
Could this be not static ???

This would prevent 2 instances in the same page to require the same setup. This would follow the same than textureFormatToUse

Reply:
I don't think it would require two instances. This array is populated but the `setCompressedTextureExclusions` and only needs to be called once. Can you explain?

Reply:
Yes but setCompressedTextureExclusions is calling into ThinEngine.ExcludedCompressedTextures = skippedFiles;

meaning for any different instance of Engine (if I have several canvases in my page and each of them have different engine) the value will be the same.

In the set method you want to use this.excludedCompressedTextures = skippedFiles;

so all of them can use a different one. Think about it like this one has been done
```
// Hardware supported Compressed Textures
protected _texturesSupported = new Array<string>();
```

Reply:
Ah, I see now. Ok, makes more sense, thank you!

Reply:
Done!